src/runtime_config/default_actions.js

```diff
@@ -250,32 +168,11 @@ const actions = new Map([
       }
     ),
   ],
-  [
-    "valid_pem",
-    new Action(
-      function (args) {
-        checkX509CertChain(args.pem, "pem");
-      },
-      function (args) {}
-    ),
-  ],
-  [
-    "always_throw_in_apply",
-    new Action(
-      function (args) {
-        return true;
-      },
-      function (args) {
-        throw new Error("Error message");
-      }
-    ),
-  ],
   [
     "set_ca_cert_bundle",
```

Comment:
Note, this doesn't quite match the spec, where we didn't have _bundle: https://github.com/microsoft/CCF/discussions/2169#discussioncomment-373490

I've left it as is because it seems reasonable anyway, @letmaik feel free to correct the other way if you prefer.

Reply:
I've changed it in the spec.